module/userProjects/helper.js

```diff
@@ -3577,6 +3627,57 @@ module.exports = class UserProjectsHelper {
 			}
 		})
 	}
+
+	/**
+	 * update project infromation
+	 * @method
+	 * @name update
+	 * @param {Object} filter - filter to search project to be updated.
+	 * @param {Object} updateTo - data which will be updated.
+	 * @param {String} userId - Logged in user Id.
+	 * @param {String} userToken - filter to search project to be updated.
+	 * @param {String} [appName = ""] - App Name.
+	 * @param {String} [appVersion = ""] - App Version.
+	 * @returns {Object} status of update project
+	 */
+
+	static update(filter, updateTo, userId, userToken, appName = '', appVersion = '') {
+		return new Promise(async (resolve, reject) => {
+			try {
+				const userProject = await projectQueries.projectDocument(filter, ['_id', 'reflection', 'tasks'])
+				updateTo.tasks = _fillMissingTaskInformation(updateTo.tasks, userProject[0].tasks)
+				let updateResult = await this.sync(
+					filter._id,
+					'',
+					updateTo,
+					userId,
+					userToken,
+					appName,
+					appVersion,
+					true
+				)
+
+				if (updateResult.message == CONSTANTS.apiResponses.USER_PROJECT_UPDATED) {
+					return resolve({
+						message: CONSTANTS.apiResponses.USER_PROJECT_UPDATED,
+						success: true,
+						result: {
+							_id: filter._id,
+						},
+					})
+				} else {
+					throw new Error(CONSTANTS.apiResponses.PROJECT_UPDATE_FAILED)
```

Comment:
@borkarsaish65 Please throw error with proper message and status

Reply:
done